osidb/models.py

```diff
@@ -840,6 +842,28 @@ def _validate_cwe_format(self):
     #         if self.unembargo_dt < datetime.now():
     #             self.embargoed = False
 
+    def save(self, *args, **kwargs):
+        """
+        save with restrictions
+        """
+        # placeholder flaws have a special handling mainly in sense of visibility
+        # while we deprecate this concept so let us restrict the write changes
+        if self.is_placeholder:
+            raise NotImplementedError(
+                "OSIDB does not support write operations on placeholder flaws"
+            )
+
+        super().save(*args, **kwargs)
+
+    @property
+    def is_placeholder(self):
+        """
+        placeholder flaws contain a special Bugzilla keyword
+        """
+        return FLAW_PLACEHOLDER_KEYWORD in json.loads(
+            self.meta_attr.get("keywords", "[]")
+        )
+
```

Comment:
This effectively means that updates of placeholder flaws from Bugzilla will not succeed, at that point isn't it better to simply discard them in bzimport?

Basically I'm not sure what's the goal of these "placeholder flaws" in OSIDB, do we want to track them at all?